### ISSUE TITLE
Support Dark Mode

### ISSUE THREAD

**Issue by ylor** [johman10/flood-for-transmission#6]
Please consider implementing a dark mode using the CSS media query for `prefers-color-scheme`.

Example:
```css
@media (prefers-color-scheme: dark) {
  /* dark mode goes here! */
}
```

Thanks for making this and extra points for using Svelte 😍!

**Comment by johman10** [comment]
Hi! Thanks for your input, I'm totally up for trying. I'm working on multi torrent adding right now, but this will be my next feature then!

Will let you know here when it's done!

**Comment by johman10** [comment]
I just wanted to let you know that this in work in progress. I'm working on using css variables for all colors to make this easier. I hope to soon deliver what you're asking for, almost all colors have been abstracted away now. :)

**Comment by ylor** [comment]
No worries and no rush. Thank you for your time.

**Comment by johman10** [comment]
I created a branch with my extraction work: https://github.com/johman10/flood-for-transmission/tree/feature-dark-mode

No dark mode there yet, but I will merge this branch into master when I feel like it's ready.

If you're curious feel free to checkout the branch (`git checkout feature-dark-mode`) and then follow the [update steps in the readme](https://github.com/johman10/flood-for-transmission#updating) to test it. :)
Do let me know here if anything seems off and I will have a look at it. :)

**Comment by johman10** [comment]
I just released a dark mode in #76. Check it out for a sneak peek.

I know it might not be perfect, but I had a hard time deciding on dark colors. I figured to just go with something and see what everyone things. Please bring back feedback/color suggestions where ever you feel it could be useful.

Thanks for the feature request, the patience, and the great suggestion. Hope this helps everyone!
